gobblin-completeness/src/main/java/org/apache/gobblin/completeness/verifier/KafkaAuditCountVerifier.java

```diff
@@ -171,16 +171,19 @@ private double calculateClassicCompleteness(String datasetName, long beginInMill
     for (String refTier: this.refTiers) {
```

Comment:
Trying to get more clarity here
Current Scenario:
source tier = gobblin, ref tier = agg cluster 

What are the refTiers in the new scenario is it (agg cluster, local cluster1) or (agg cluster, local cluster1,local cluster2,local cluster3 ) and what is the expected behavior?

Reply:
It's not necessarily for local tier, but in general to support multiple reference tiers. As for the local stuff as it's internal specific detail, reply with you offline

Reply:
I think better way to put it is : we wait until src tier catches up to all reference tiers (upto 00.9%)